lib/components/SelectScreens.js

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Text, Header } from '@bpanel/bpanel-ui';
+import { selectStep as style } from '../style';
+
+class SelectStep extends PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  /* selection schema
+   * [{
+   *    header: '',
+   *    text: '',
+   *    icon: '',
+   *    click: ['', '']
+   * }]
+   */
+  static get defaultProps() {
+    return {
+      selection: [],
+    };
+  }
+
+  static get propTypes() {
+    return {
+      selection: PropTypes.array,
+      select: PropTypes.func,
+    };
+  }
+
+  render() {
+    const { selection } = this.props;
+    let { select } = this.props;
+    let itemContainer = style.itemContainer;
+    return (
+      <div className="row" style={style.container}>
+        {selection.map((s, i) => {
+          let onclick = select;
+          // buttons can be disabled
+          // change onclick to noop and style container as disabled
+          if (!s.enabled) {
+            itemContainer = Object.assign(
```

Comment:
I really hate the way the linter made this

Reply:
ah I see. I think you could use destructuring here which would reduce the items and probably not need the line split anymore. e.g. ` {...style.itemContainer, ...styled.disabled}`